master/buildbot/test/util/decorators.py

```diff
@@ -40,9 +40,13 @@ def wrap(func):
     return wrap
 
 
-def flaky(bugNumber=None, issueNumber=None):
+def flaky(bugNumber=None, issueNumber=None, onPlatform=None):
     def wrap(fn):
-        if not os.environ.get(_FLAKY_ENV_VAR):
+        match_platform = True
```

Comment:
```suggestion
     if onPlatform is not None and sys.platform != onPlatform:
         return fn
```

Reply:
Agreed. Wanted to preserve the present style of the function.